app/javascript/flavours/glitch/reducers/compose.js

```diff
@@ -148,8 +148,12 @@ function continueThread (state, status) {
     text = text + apiStatusToTextHashtags(state, status);
     map.set('text', text);
     if (status.spoiler_text) {
+      let spoiler_text = status.spoiler_text;
+      if (!spoiler_text.match(/^re[: ]/i)) {
+        're: '.concat(status.spoiler_text);
+      }
       map.set('spoiler', true);
-      map.set('spoiler_text', status.spoiler_text);
+      map.set('spoiler_text', spoiler_text);
```

Comment:
This part is exclusively for self-replies in threaded mode. I think this makes less sense to do it here than on actual replies.
The part for actual replies is here: https://github.com/glitch-soc/mastodon/pull/728/files#diff-879b0d35ea81262bbc03aecaff904051L316

Reply:
thanks, this should be fixed now!